Mark non-grouped convolutions instances as deprecated (#2595)
ROCm/composable_kernel: CHANGELOG.md

None

### Upcoming changes

> * Non-grouped convolutions will be deprecated. All of their functionality is supported by grouped convolution.

Reviewer: ```suggestion
* Non-grouped convolutions are deprecated. All of their functionality is supported by grouped convolution.
```
